setup.py

```diff
@@ -44,11 +44,13 @@
     'openfisca-extension-template >= 1.2.0rc0, < 2.0.0',
     'pycodestyle >= 2.8.0, < 2.9.0',
     'pylint == 2.10.2',
+    'pylint-pytest == 1.1.2',
+    'pyupgrade == 2.28.0',
```

Comment:
Do not pin versions.

Reply:
Even if not pinned, some constraint needs to be in place, otherwise cache will never update them, and we will end with errors like the one we have now in CI.